test/user-profile/e2e/user-profile.e2e.spec.js

```diff
@@ -7,9 +7,11 @@ module.exports = E2eHelpers.createE2eTest(
     let token = LoginHelpers.getUserToken();
 
     LoginHelpers.logIn(token, client, '/profile', 1)
-      .waitForElementVisible('.loading-indicator-container', Timeouts.normal)
-      .waitForElementNotPresent('.loading-indicator-container', Timeouts.slow)
-//      .waitForElementVisible('.usa-button-big', Timeouts.normal)
+      // The loading indicator tests seem to be problematic because the thing goes away fairly quickly
```

Comment:
In this particular case, it seems like the test is not really testing much, right? It's also not really a sauce thing... the test is flawed in that it requires the timing to work in a specific way, so since it's (IMO) both useless and flawed it seemed worth removing.